app/models/automation_request.rb

```diff
@@ -27,7 +27,7 @@ def self.create_from_ws(version, user, uri_parts, parameters, requester)
     options[:class_name]    = (options.delete(:class) || DEFAULT_CLASS).strip.gsub(/(^\/|\/$)/, "")
     options[:instance_name] = (options.delete(:instance) || DEFAULT_INSTANCE).strip
     options[:schedule_type] = parameters['schedule_time'].present? ? "schedule" : "immediately"
-    options[:schedule_time] = parameters['schedule_time'] if parameters['schedule_time']
+    options[:schedule_time] = parameters['schedule_time'].to_i.days.from_now if parameters['schedule_time']
```

Comment:
I hope all clients of this code realizes `schedule_time` is in days... it's not obvious  😿